website/source/docs/auth/gcp.html.md

```diff
@@ -0,0 +1,339 @@
+---
+layout: "docs"
+page_title: "Auth Backend: GCP"
+sidebar_current: "docs-auth-gcp"
```

Comment:
This page needs to be added to the layouts as well.

File: `source/layouts/docs.erb`
Section: `docs-auth`
Content:
```
<li<%= sidebar_current("docs-auth-gcp") %>>
            <a href="/docs/auth/gcp.html">GCP</a>
</li>
```